Workflow.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 23 12:30:17 2016
+
+@author: mina
+"""
+
+import yaml
+
+class Workflow:
+    def __init__(adress,self):      
```

Comment:
`self` should be the first argument here and I also think the workfow config file parameter could be more descriptive, maybe something like `workflow_config_file`.

```
def __init__(self, workflow_config_file):
```
```